independent-publisher-connectors/HubSpot CRM/apiProperties.json

```diff
@@ -0,0 +1,75 @@
+{
+  "properties": {
+    "connectionParameterSets": {
+      "uiDefinition": {
+        "displayName": "Authentication Type",
+        "description": "Type of authentication to be used."
+      },
+      "values": [
+        {
+          "name": "hubspot-oauth",
+          "uiDefinition": {
+            "displayName": "Use HubSpot OAuth 2.0",
+            "description": "Log in using HubSpot app client ID and secret."
+          },
+          "parameters": {
+            "token": {
+              "type": "oauthSetting",
+              "oAuthSettings": {
+                "identityProvider": "oauth2",
+                "clientId": "[Your_Value]",
+                "scopes": [
+                  "crm.objects.marketing_events.read contacts content reports actions timeline oauth forms files crm.objects.marketing_events.write crm.schemas.custom.read crm.objects.custom.read crm.objects.feedback_submissions.read crm.objects.custom.write": "crm.objects.marketing_events.read contacts content reports actions timeline oauth forms files crm.objects.marketing_events.write crm.schemas.custom.read crm.objects.custom.read crm.objects.feedback_submissions.read crm.objects.custom.write"

```

Comment:
Please correct col 254. The scopes are repeated.

Reply:
Fixed